trackers/run_tracker.py

```diff
@@ -72,26 +85,62 @@ def run(global_config_file, dash_schema_file, pipelines, run_id=1):
                         status = func(subject_dir, session_id, run_id)
                         # print(f"task_name: {name}, status: {status}")
                         _df.loc[bids_id,name] = status
-                        _df.loc[bids_id,"pipeline_starttime"] = get_start_time(subject_dir)
-                        _df.loc[bids_id,"pipeline_endtime"] = UNAVAILABLE # TODO
+                    _df.loc[bids_id,"pipeline_starttime"] = get_start_time(subject_dir)
+                    _df.loc[bids_id,"pipeline_endtime"] = get_end_time(subject_dir)
+                    n_participants += 1
                 else:
                     # print(f"Pipeline output not found for bids_id: {bids_id}, session: {session}")
                     for name in status_check_dict.keys():                    
                         _df.loc[bids_id,name] = UNAVAILABLE
-                        _df.loc[bids_id,"pipeline_starttime"] = UNAVAILABLE
-                        _df.loc[bids_id,"pipeline_endtime"] = UNAVAILABLE
+                    _df.loc[bids_id,"pipeline_starttime"] = UNAVAILABLE
+                    _df.loc[bids_id,"pipeline_endtime"] = UNAVAILABLE
+
+                # don't check all participants if testing
+                if testing and n_participants > N_TESTING:
+                    break
+
+            proc_status_session_dfs.append(_df)
 
-            proc_status_dfs.append(_df)
+        # new rows for this pipeline
+        pipeline_proc_status_df = pd.concat(proc_status_session_dfs, axis='index').reset_index(drop=True)
 
-    proc_status_df = pd.concat(proc_status_dfs, axis='index')
+        # add old rows from other pipelines and sort for consistent order
+        proc_status_df: pd.DataFrame = pd.concat([old_proc_status_df, pipeline_proc_status_df], axis='index')
+        proc_status_df = proc_status_df.sort_values(["pipeline_name", "pipeline_version", "bids_id"], ignore_index=True)
```

Comment:
It doesn't matter (much) if we sort by `"participant_id"` or `"bids_id"` since it should be a one-to-one mapping anyway and this is just so that the order is consistent between pipelines